community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -752,7 +752,7 @@ object projects:
 
   lazy val http4s = SbtCommunityProject(
     project = "http4s",
-    sbtTestCommand = """set ThisBuild / tlFatalWarnings := false; server/test; client/test; ember-core/test; ember-server/test; ember-client/test; circe/test""",
+    sbtTestCommand = """set ThisBuild / tlFatalWarnings := false; rootJVM/test""",
```

Comment:
We slimmed down the http4s repo (it was previously a monorepo). So we should no longer need to pick-and-choose specific modules.